### PR TITLE
Feature/37

### PR DESCRIPTION
**Pull request by DPDeepak**
Ticket: https://github.com/nodexpertsdev/RnD/issues/37

### CODE REVIEW

**Comment by manish-nx** [comment]
Look closely, you are going to merging it to Master. Please change the base branch to develop

**Comment by DPDeepak** [comment]
> Look closely, you are going to merging it to Master. Please change the base branch to develop

Fixed it.

File: expressEngine/model/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import User from './collection/user';
 import Order from './collection/order';
-import Product from './collection/product';
+import ProductModel from './collection/product';
 
 export {
   User,
   Order,
-  Product,
+  ProductModel,
```
Comment (manish-nx) [inline expressEngine/model/index.js]: Look at the naming of other models.
It's User and Order.
So please change ProductModel to Product and also change wherever it is imported as ProductModel.

Comment (DPDeepak) [reply expressEngine/model/index.js]: Done! Please have a look.

File: expressEngine/product/service/product.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import { BaseService, DBService } from '../../lib/service/index';
 
 // import collections
-import { ProductModel, ProductDetail } from '../../model/index';
+import { Product, ProductDetail } from '../../model/index';
```
Comment (aman-successive) [inline expressEngine/product/service/product.js]: Remove index from import statement

**Pull request merged**